process.go

```diff
@@ -517,6 +517,23 @@ func (p *process) getChanInfo(channel uint64) (*channel, error) {
 		return ch, nil
 	}
 
+	// If the channel is not open, fall back to checking our closed
+	// channels.
+	closedChannels, err := p.client.listClosedChannels()
+	if err != nil {
+		return nil, err
+	}
+
+	// Add to cache and try again.
+	for chanId, ch := range closedChannels {
+		p.chanMap[chanId] = ch
+	}
+
+	ch, ok = p.chanMap[channel]
+	if ok {
+		return ch, nil
+	}
```

Comment:
Do the pending (close) channels need to be added in here too?

Then I think it would be impossible in theory for the channel to still not be found, in which case a log message is warranted?

Reply:
Going to need https://github.com/lightningnetwork/lnd/pull/8000 for short channel ID in pending channels. 

Open to suggestions on how to handle in the meantime! I think error out is _okay_, could also downgrade to a log.

Reply:
Yes, let's leave this behavior as is.